csv_export.php

```diff
@@ -45,6 +45,11 @@
 require_api( 'helper_api.php' );
 require_api( 'print_api.php' );
 
+if( !access_has_project_level( config_get( 'export_issues_threshold' ) ) ) {
+	# current user is not allowed to export content => redirection (compliant with bug_count = 0)
+	print_header_redirect( 'view_all_set.php?type=0' );
```

Comment:
When user is not allowed to export, the displayed page is blank.
This redirect is the workaround used here, as it is done by csv_export when 0 bug has to be exported (see row 71).